projects/extension/src/background/ConnectionManager.ts

```diff
@@ -15,6 +15,17 @@ import { StateEmitter, State } from "./types"
 import { NetworkMainInfo, Network } from "../types"
 import { logger } from "@polkadot/util"
 import { ToExtension } from "@substrate/connect-extension-protocol"
+import westend from "../../public/assets/westend.json"
+import kusama from "../../public/assets/kusama.json"
+import polkadot from "../../public/assets/polkadot.json"
+import rococo from "../../public/assets/rococo.json"
+
+export const relayChains: Map<string, string> = new Map<string, string>([
```

Comment:
```suggestion
export const wellKnownChains: Map<string, string> = new Map<string, string>([
```

I know that in practice well known chains are always relay chains, but they're still separate concepts

Reply:
totally!